app/js/actions/activeOrderAction.js

```diff
@@ -18,8 +17,8 @@ const activeOrderActionError = error => ({
 });
 
 const activeOrderAction = (careSetting, patientUuid) => (dispatch) => {
-  dispatch(loading('FETCH_ACTIVE_ORDER', true));
-  return axios.get(`${apiBaseUrl}/order?careSetting=${careSetting}&patient=${patientUuid}&t=drugorder&v=full`, {
+  dispatch(loading(true));
```

Comment:
Any reason for replacing the existing code on line 20?

Reply:
I did not intend to remove it. Have added it back, thank u